### ISSUE TITLE
Self-generated certificate is regenerated every 15 seconds

### ISSUE THREAD

**Issue by bgranvea** [containous/traefik#5557]
### Do you want to request a *feature* or report a *bug*?

Bug

### Did you try using a 1.7.x configuration for the version 2.0?

- [ ] Yes
- [X] No

### What did you do?
I configured Traefik with Docker provider and swarmMode=true, and I deployed a service with an HTTPS entrypoint and no tls options.

### What did you expect to see?
Traefik should generate a certificate for HTTPS and keep it, so that it can be added to trusted certificates in a browser.

### What did you see instead?
Traefik generates a new certificate for HTTPS every 15 seconds, so the browser keeps telling that the certificate is not trusted.

### Output of `traefik version`: (_What version of Traefik are you using?_)
```
Version:      2.0.1
Codename:     montdor
Go version:   go1.13.1
Built:        2019-09-26T16:18:03Z
OS/Arch:      linux/amd64
```

### If applicable, please paste the log output in DEBUG level (`--log.level=DEBUG` switch)

```
time="2019-09-30T13:54:52Z" level=debug msg="No default certificate, generating one"
...
time="2019-09-30T13:55:07Z" level=debug msg="No default certificate, generating one"
...
time="2019-09-30T13:55:22Z" level=debug msg="No default certificate, generating one"
```

**Comment by mmatur** [comment]
Hi @bgranvea ,

Thanks for your interest in the project. 

It seems to be a duplicate of #5381

So I will close this issue.

**Comment by bgranvea** [comment]
hum yes maybe it is the same root cause as #5381 but with swarmMode=true, it seems even worse as the certificate is regenerated even when no new container is deployed.

And I don't understand why you don't consider #5381 as a bug: IMHO the self-generated certificate feature is not usable with a browser.

**Comment by dduportal** [comment]
Hi @bgranvea, for the "swarmMode" making this behavior even worse, it's related to the issues #5419. This issue had been closed and will be in the next `2.0.x` release. You can already test it with the image `containous/traefik:experimental-v2.0` if you want to live on the edge :)

As for #5381, we consider the "self generated certificate" as a "degraded" mode. If you're willing to use a self signed certificate for all of your services, we recommend generating one and setting Traefik to use it as explained in <https://docs.traefik.io/v2.0/https/tls/>. I personnaly use the awesome tool `mkcert` https://github.com/FiloSottile/mkcert which generate for multiple domains and wildcards including `*.local`, and install the CA in my browsers.
